src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+export const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, { payload }) => {
+      // For adding the book to the bookstore
+      state.books.push(payload.book);
+    },
+    removeBook: (state, { payload }) => ({
+      // Remove the book from the bookstore
+      books: state.books.filter((book) => book.id !== payload.id),
+    }),
```

Comment:
- [ ] 🙏Please correct this `remove` action since the reducer is the function. It should look like this:
```suggestion
   removeBook: (state, { payload }) => {
      // Remove the book from the bookstore
      state.books = state.books.filter((book) => book.id !== payload.id);
    },
```

Reply:
Thank you for the suggestion @abdallahmalima 🙏